materialize-boilerplate/testdata/generate-spec-proto.sh

```diff
@@ -28,11 +28,9 @@ function cleanup {
 }
 trap cleanup EXIT
 
-flowctl-go api build \
-    --build-id build-id \
-    --build-db ${TEMP_DIR}/build.db \
-    --network flow-test \
-    --source $1 \
-    --log.level info
+flowctl raw build \
```

Comment:
Moving over to `flowctl raw build` for long-term compatibility. This did cause all of the generate proto files to be updated.